src/styles/main.scss

```diff
@@ -1,3 +1,20 @@
-body {
```

Comment:
Add CSS reset or normalize.css before importing other styles to ensure consistent styling across different browsers.